media-libs/openjpeg/openjpeg-2.1.2.recipe

```diff
@@ -85,5 +86,7 @@ INSTALL()
 
 TEST()
 {
-	make check
+	cd tests
+	cmake ..
+	make
```

Comment:
Nope, IIRC the generated Makefile does not have a "check" target.

But thanks a lot for your pointers. I think the project has a CTestTestfile.cmake file in a subdirectory, so I guess that calling ctest from within that directory will do the trick. I'll try that and update the recipe with an improved TEST function.